### ISSUE TITLE
[SensioFrameworkExtraBundle 4.0] Template missing when using new snake_case format

### ISSUE THREAD

**Issue by ruudk** [Haehnchen/idea-php-symfony2-plugin#1296]
SensioFrameworkExtraBundle 4.0+ introduced a BC break:
```
changed template name generation from camelCase to under_score for both files and directories
```

This causes the plugin to no longer find the templates.

I tried looking into the source where this might happen, maybe you can point me in the right direction so that I can try to fix it?

My controller `AdminBundle\Controller\Financial\ShowPaymentController::__invoke`
My template: `src/AdminBundle/Resources/views/financial/show_payment.html.twig`

**Comment by ruudk** [comment]
Seems to be a duplicate of https://github.com/Haehnchen/idea-php-symfony2-plugin/issues/1082 that has been open for a long time. 

I'd like to have a look to see if we can fix this, but need some help to find the place.

**Comment by TomasVotruba** [comment]
Fixed by #1389
